controllers/notification/mailgun_notification_delivery_service.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/mailgun/mailgun-go/v4"
 )
 
+const (
+	ContextReplyTo = "ReplyTo"
```

Comment:
@sbryzak so the resulting spec will look like:
```yaml
spec:
  context:
    ReplyTo: some-reply@dummy.com
```

shouldn't the `ReplyTo` start with lower-case `r`?

Reply:
it's just a detail, of course - I mentioned it because all fields in our CRs start with lower-case, so it would be maybe good to follow that pattern :-).

Reply:
That would be inconsistent with our other template variable naming.  Take a look through the notification templates [here](https://github.com/codeready-toolchain/host-operator/tree/master/deploy/templates/notificationtemplates) for other examples.

Reply:
I forgot about that - thanks for letting me know. :+1: